pkg/middleware/auth_proxy.go

```diff
@@ -12,13 +14,18 @@ const (
 	cachePrefix = authproxy.CachePrefix
 )
 
+var header = setting.AuthProxyHeaderName
+
 func initContextWithAuthProxy(store *remotecache.RemoteCache, ctx *m.ReqContext, orgID int64) bool {
+	username := ctx.Req.Header.Get(header)
 	auth := authproxy.New(&authproxy.Options{
 		Store: store,
 		Ctx:   ctx,
 		OrgID: orgID,
 	})
 
+	logger := log.New("auth proxy")
```

Comment:
the logger name should not have white space (otherwise the level filters you specify in grafana.ini wont work) you dots instead to signify hierarchy

Reply:
Heh, that's why we don't use the spaces :)